packages/cozy-sharing/src/components/ShareDialogCozyToCozy.jsx

```diff
@@ -32,76 +159,148 @@ const ShareDialogCozyToCozy = ({
   showShareByEmail,
   showShareByLink,
   showShareOnlyByLink,
-  showWhoHasAccess
+  showWhoHasAccess,
+  hasTwoStepsConfirmation = false,
+  twoStepsConfirmationMethods = {}
 }) => {
   const { t } = useI18n()
+
+  const [status, setStatus] = React.useState('loading')
```

Comment:
So by default we're displaying the spinner and then make a re-render to display the modal. It's not so much, but can it be avoided somehow?

Reply:
Would you prefer if I set `sharing` by default? Or maybe a `blank` state?

Reply:
Can't we detect that we have to fetch something? Or 

```
useState(hasTwoStepsConfirmation &&
      twoStepsConfirmationMethods?.getRecipientsToBeConfirmed ? 'loading' : 'loaded') 
      ```

Reply:
Proposition: https://github.com/cozy/cozy-libs/pull/1340/commits/e6549ee80dfde4a32146129da7b48b0611e1e125